src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.cs

```diff
@@ -68,7 +68,8 @@ public static bool TryLoad(string libraryPath, out IntPtr handle)
         /// If DllImportSearchPath parameter is non-null, the flags in this enumeration are used.
         /// Otherwise, the flags specified by the DefaultDllImportSearchPaths attribute on the
         /// calling assembly (if any) are used.
-        /// This LoadLibrary() method does not invoke the managed call-backs for native library resolution:
+        /// This method behaves as if DllImport was used in the specified <paramref name="assembly"/>.
+        /// It will invoke the managed call-backs for native library resolution:
```

Comment:
Please add a comment here that the recursive invocation is guarded, so that his method is safe to use within these callbacks/events. This is helpful because these XML comments are picked up by the documentation.
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.nativelibrary.tryload?view=netcore-3.1